pkg/backend/backend.go

```diff
@@ -189,6 +189,17 @@ type Backend interface {
 	CurrentUser() (string, error)
 }
 
+// SpecificDeploymentExporter is an interface defining an additional capability of a Backend, specifically the
+// ability to export a specific versions of a stack's deployment. This isn't a requirement for all backends and
+// should be checked for dynamically.
+type SpecificDeploymentExporter interface {
```

Comment:
Tiny nit for consistency with the method name
```suggestion
type VersionedDeploymentExporter interface {
```